### ISSUE TITLE
Sublime AVR build issues

### ISSUE THREAD

**Issue by rishikanthc** [kblomqvist/SublimeAVR#5]
Hi,

 I followed the procedure for installation of the plugin. However when i compile or build the project, it generates "clang: warning: treating 'c' input as 'c++' when in C++ mode, this behavior is deprecated
/Users/Rishikanth/Documents/EmbeddedC/main.c:1:10: fatal error: 'avr/io.h' file not found
#include <avr/io.h>
         ^
1 error generated."

It seems to be building the file as a normal C++ file instead of using avr-gcc.

Please help me resolve this.

**Comment by kblomqvist** [comment]
Hmm, it may be that the SublimeClang plugin passes header files for the clang parser and this is now deprecated. http://stackoverflow.com/questions/23534362/clang-warning-treating-c-header-input-as-c-header-when-in-c-mode-this

However, I don't understand how this can happen with the clang which is provided among the SublimeClang (and hence SublimeAVR). Do you have clang installed separately?

**Comment by maxlazarus** [comment]
Hello, thanks for making this package, I am having issues getting it to work however.

When I choose to create a project no dialogue pops up.  The bottom bar shows 'Sublime AVR: installing the dependency package Sublime Clang... clang ok " the first time I try to create a project after opening Sublime and then nothing else happens.  On subsequent attempts to create a project there is no response unless I close and reopen Sublime, where the same things happens.

I'm using Sublime Text build 3083 with OSX 10.9.5

Thanks in advance

**Comment by kblomqvist** [comment]
@maxlazarus Maybe it's related to #4. Can you follow the steps there to debug the issue?

**Comment by maxlazarus** [comment]
I looked at that but didn't find anything that helpd, the package can find avr-gcc fine in the path "/usr/local/CrossPack-AVR-20131216/bin/" and I can see all the devices when I run avr-as, there are no error messages popping up or anything, just does not open any prompt.

**Comment by kblomqvist** [comment]
Could you still try to print devices variable in `avrgcc.py` to check that the list is not empty.

**Comment by maxlazarus** [comment]
Hello again, I don't have access to the OSX machine that I installed AVRsublime on right now but I will get that py variable and update here when I do have access.

I installed sublimeAVR on a Windows machine instead and got a bit farther.  The menu pops up, I can select an MCU and name a project which creates a file folder and filesystem including the main.c and other files BUT it does not create a project file, so I cannot open the project.

Thank you.
